### PR TITLE
Update training snakes

### PR DESCRIPTION
**Pull request by john-swu**

### CODE REVIEW

File: server/src/rest/snakesRouter.ts [hunk 1]
```diff
@@ -6,15 +6,15 @@ import * as _ from 'lodash'
...
+    const snakes = _.range(0, 11).map(i => {
         return {
-            name: `SWU Training Snake ${i}`,
+            name: `Training Snake ${i}`,
             url: `https://battlesnake-training-snakes.herokuapp.com/snake_${i}`
         }
     })
 
     snakes.push({
-        name: "DSnek",
```
Comment (daniel-swu) [inline server/src/rest/snakesRouter.ts]: Why the rename?

Comment (john-swu) [reply server/src/rest/snakesRouter.ts]: to make it seem consistent, as if they are all training snakes. seemed odd to have one differently named.

**Comment by daniel-swu** [comment]
We're going to have the bounty snakes in there too. So it won't be the only
one. I don't want my snake labelled a training snake

File: server/src/rest/snakesRouter.ts (hunk 1 above)
Comment (john-swu) [reply server/src/rest/snakesRouter.ts]: sorry, my mistake. undoing.

**Comment by john-swu** [comment]
@daniel-swu I forgot, in my head it's split pages for some reason, sorry.
Undid the rename.

**Pull request merged**